hexa/pipelines/graphql/schema.graphql

```diff
@@ -85,6 +85,7 @@ type PipelineRun {
   code: String!
   sendMailNotifications: Boolean!
   timeout: Int
+  datasetVersion: [DatasetVersion!]!
```

Comment:
I'm not a fan of datasetVersion but I cannot think of another name... (Let's put it in plural)